mission-racing/src/main/java/com/racing/controller/RacingController.java

```diff
@@ -19,24 +22,28 @@ public RacingController(RacingService racingService) {
     }
 
     @PostMapping
-    public void registerCars(@RequestBody CarRegister register) {
-        racingService.registerCars(register);
+    public ResponseEntity<Void> registerCars(@RequestBody CarRegisterRequest registerRequest) {
+        racingService.registerCars(registerRequest);
+        return ResponseEntity.ok().build();
     }
 
     @PostMapping("/racing")
-    public void startRace() {
+    public ResponseEntity<Void> startRace() {
         racingService.race();
+        return ResponseEntity.ok().build();
     }
 
     @GetMapping
     public ResponseEntity<RaceResult> getRaceResults() {
-        RaceResult resultResponse = racingService.getRaceResults();
+        Cars cars = racingService.getCarResult();
+        RaceResult resultResponse = RaceResult.resultResponse(cars);
         return ResponseEntity.ok(resultResponse);
```

Comment:
```suggestion
        return ResponseEntity.ok(RaceResult.resultResponse(cars));
```

선택 사항이지만 이렇게 해도 의미 파악에 문제는 없어서 좋을 것 같습니다!
별개의 리뷰지만 정적 팩토리 메서드에서 메서드 명은 컨벤션은 아니지만 보통 많은 사람들이 of, from, with ... 라는 네이밍을 많이 사용하곤 합니다.

1. of : 정적 팩토리 메서드에서 매개변수가 여러 개인 경우 (적합한 예시로는 자바에서 List.of(1,2,3,4,5)가 있겠네요!)
2. from : 매개변수가 한 개인 경우 (ex. RaceResult.from(cars))
3. 동사 + with : 구체적인 뜻을 부여해주고 싶은 경우 (Car.createWithHandle(...))